### DISCUSSION TITLE
posts/scrypted-home-hub/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#283] [Comments]
# posts/scrypted-home-hub/

Meet Scrypted an Open Source app that will let you connect almost any camera to any home hub, certified or not! You can connect popular devices from UniFi, Amcrest, Hikvision, Nest & Google, Tuya, Reolink, and many others to your home hub of choice, whether that be Apple’s HomeKit, Google Home, Alexa, or even Home Assistant.This lets you choose and reuse your own devices and take advantage of the automation and integration you get with your Smart Home Hub.

https://technotim.live/posts/scrypted-home-hub/

**Comment by dr01dy** [comment]
Good break down! 

Wondering if you had any firewall issues with multicast and rstp to get the streams exposed in order for the HomeKit paring to start working?

**Comment by inspectorgit** [comment]
Hey Tim,  I've got to ask about your scrypted setup. Are you running it on the same vlan as your cameras? I've got unifi cameras in scrypted showing but anytime I try to add them to homekit they can't be reached even with pinhole firewall rules. Would love if you could expand on your unifi firewall setup for scrypted for homekit pairing.

**Reply by timothystewart6** [reply]
Hey!   I am not, Scrypted is on a different VLAN than cameras.  You'll need to be sure that Scrypted can reach your UDM interface IP, which is where Protect is hosted.  This is a different type of firewall rule, a `LAN Local` rule (not `LAN in`)
